packages/cozy-sharing/locales/en.json

```diff
@@ -385,5 +385,9 @@
       "instruction": "For security reasons, please verify **%{userName} (%{userEmail})**'s identity. To do this, contact **%{userName}** via the means of communication of your choice (call, text message, instant messaging app...) and verify their security code.<br><br>Every Cozy user has a security code that they can find using their web application. Their security code must correspond to:",
       "instruction2": "If it is the case, then you can confirm this contact who will be considered as secure for your next sharings."
     }
+  },
+  "ConfirmRecipientModal": {
+    "title": "1 contact waiting for confirmation |||| %{smart_count} contacts waiting for confirmation",
```

Comment:
are you sure that transifex works correctly with `1 ||| smart_count` instead of `smart_count ||| smart_count`?

Reply:
Yes it works, if you look at other translations from this file this is already used like this.

Reply:
I replaced it by %{smart_count} anyway, this will avoid future confusion